lib/WP_Auth0_Lock10_Options.php

```diff
@@ -113,9 +118,14 @@ protected function build_settings( $settings ) {
 
 		$options_obj['socialButtonStyle'] = 'big';
 
-		if ( isset( $settings['gravatar'] ) && empty( $settings['gravatar'] ) ) {
+		if ( isset( $settings['gravatar'] ) && '' !== $settings['gravatar'] && empty( $settings['gravatar'] ) ) {
 			$options_obj['avatar'] = null;
 		}
+
+		if ( ! empty( $settings['gravatar'] ) ) {
```

Comment:
you check for `empty` but not for `isset`, why ?

Reply:
`empty()` covers the `isset()` case. Means "no truth-y value to use".